source/change-streams/change-streams.rst

```diff
@@ -546,16 +546,16 @@ so resuming is impossible.
 Why do we need to send a default ``startAtOperationTime`` in the ``$changeStream`` stage?
 -----------------------------------------------------------------------------------------
 
-``startAtOperationTime`` allows a user to create a change stream that does not return
-results until a later point in time. However, it has the added benefit of allowing
-a change stream to resume and receive all changes that it may have missed since it's first
-creation.
+``startAtOperationTime`` allows a user to create a resumable change stream even when a result
+(and corresponding resultToken) is not available until a later point in time. However, 
```

Comment:
I think the second sentence "However, ..." is either unnecessary or can be streamlined.